### ISSUE TITLE
[Accessibility] Rating Scale: Screen readers do not announce the question title when the question is focused using the keyboard

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#10117]
T24027 - Screenreader issues continued
https://surveyjs.answerdesk.io/internal/ticket/details/T24027
***
Please test the following survey using JAWS/NVDA.
```
{
  "pages": [
    {
      "name": "page1",
      "elements": [
        {
          "type": "text",
          "name": "question1"
        },
        {
          "type": "rating",
          "name": "satisfaction-numeric",
          "title": "How satisfied are you with our product?",
          "description": "Numeric rating scale",
          "autoGenerate": false,
          "rateCount": 10,
          "rateValues": [
            1,
            2,
            3,
            4,
            5,
            6,
            7,
            8,
            9,
            10
          ]
        }
      ]
    }
  ]
}
```
Issue: When a Rating scale question is focused using the keyboard (TAB), the question title is not announced. NVDA announces it as `1  radio button  not checked  1 of 10`.
Expected behavior: a question title is announced when a rating scale receices the focus. For instance, when a Text question is focused, the NVDA logs `question1  edit  blank`.

**Comment by moulichan99** [comment]
I'm a Senior Frontend Engineer with over 5 years of experience, and I'm new to the open-source community. I'd love to contribute — may I work on this issue?

**Comment by dmitry-kurmanov** [comment]
> I'm a Senior Frontend Engineer with over 5 years of experience, and I'm new to the open-source community. I'd love to contribute — may I work on this issue?

The issue is already closed, thanks!
